app/services/message-service.js

```diff
@@ -64,6 +53,16 @@ export const buildOutboundMessage = (messageId, inboundMessage) => {
   }
 }
 
+const sendMessageToSfd = async (logger, outboundMessage) => {
+  try {
+    await sendSfdMessageRequest(outboundMessage)
+    return { success: true }
+  } catch (error) {
+    logger.error(`Failed to send outbound message to single front door. ${error.message}`)
```

Comment:
think we should log (or set) the actual error

Reply:
Agreed, will do now